configuration/index.md

```diff
@@ -44,98 +47,132 @@ Every interface has it's own strong points.
       <td>✔️</td>
-      <td>✔️</td>
-      <td>✔️</td>
+      <td>items/*.items files</td>
     </tr>
     <tr>
       <td>Define Sitemaps</td>
       <td>✔️</td>
       <td>❌</td>
       <td>❌</td>
       <td>❌</td>
+      <td>sitemaps/*.sitemap files</td>
     </tr>
     <tr>
       <td>Define Transformations</td>
       <td>✔️</td>
       <td>❌</td>
       <td>❌</td>
       <td>❌</td>
+      <td>transform/*.map files</td>
     </tr>
     <tr>
       <td>Define Persistence</td>
       <td>✔️</td>
       <td>❌</td>
       <td>❌</td>
       <td>❌</td>
+      <td>persistence/*.persist files</td>
     </tr>
     <tr>
       <td>Define Rules</td>
-      <td>✔️</td>
-      <td>✔️<br />(Experimental)</td>
-      <td>✔️<br />(Graphical)</td>
+      <td>✔️*</td>
+      <td>(❌)<br/>(only for the experimental rules engine)</td>
+      <td>(❌)<br/>(not functional)</td>
       <td>❌</td>
+      <td>rules/*.rules files</td>
     </tr>
     <tr>
-      <td>Manage Z-Wave Devices</td>
-      <td>✔️<br />(Not&nbsp;Recommended)</td>
-      <td>✔️<br />(Limited)</td>
+      <td>Manage Z-Wave Devices<br/>(see&nbsp;recommendation)</td>
+      <td>❌</td>
       <td>✔️</td>
+      <td>✔️*</td>
       <td>❌</td>
+      <td>HABmin</td>
     </tr>
     <tr>
-      <td>Modify openHAB Settings</td>
-      <td>✔️</td>
+      <td>Modify openHAB Settings/Services</td>
       <td>✔️</td>
+      <td>✔️*</td>
       <td>✔️</td>
       <td>✔️</td>
+      <td>Paper UI</td>
     </tr>
     <tr>
       <td>Install Add-ons</td>
+      <td>✔️*</td>
+      <td>✔️*</td>
       <td>✔️</td>
       <td>✔️</td>
-      <td>❌</td>
-      <td>✔️</td>
+      <td>services/addons.cfg or Paper UI</td>
     </tr>
   </tbody>
 </table>
 
-{: #text-vs-ui}
 ### Textual vs. Graphical Configuration
 
-With openHAB 1.x the smart home configuration was done via configuration files only.
-One of the most visible additions to openHAB 2 is the administrative web interface Paper UI.
-
-Things and Items can either be defined and managed in configuration files or handled by Paper UI in a [system-side database]({{base}}/administration/jsondb.html).
-
-Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a `.items` configuration file.
-
-Configuration done in files will be synced to the database but configuration done in the database will not be written to files.
+In openHAB 1.x, smart home configuration was done via configuration files only.
+openHAB 2 added the administrative web interface "Paper UI".
+It allows for configuration of the base system and setup of openHAB 2 add-ons.
+Please be aware that not every addon/binding of openHAB version 1.x has been updated to version 2 and thus to work with Paper UI.
+You can still run these but you need to use file configuration for them.
+
+Things and Items can either be defined and managed in configuration files or handled by Paper UI in a [system-side database](administration/jsondb.html).
```

Comment:
I think this link will be broken - use a server-relative link here: `[system-side database](/docs/administration/jsondb.html)`

Reply:
This will also work: `[system-side database](../administration/jsondb.html)`.